lib/facility.rb

```diff
@@ -1,14 +1,93 @@
 class Facility
-  attr_reader :name, :address, :phone, :services
+  attr_reader :name, :address, :phone, :services, :registered_vehicles, :collected_fees
 
-  def initialize(name, address, phone)
-    @name = name
-    @address = address
-    @phone = phone
+  def initialize(hash)
+    @name = hash[:name]
+    @address = hash[:address]
+    @phone = hash[:phone]
     @services = []
+    @registered_vehicles = []
+    @collected_fees = 0
   end
 
-  def add_services(service)
+  def add_service(service)
     @services << service
   end
+
+  def register_vehicle_helper(car)
```

Comment:
I'm a big fan of how you have the logic in this class organized with the use of helper methods 🎉  
This is a bit of a nitpick, but generally it's best practice to not use qualifiers like 'helper' in method names. If you wanted to polish this class even more, you could consider renaming your methods. Here are some examples/ideas:
```
#register_vehicle_helper -> #process_plate_types
#written_test_helper -> #attempt_written_test or #validate_written
#renew_drivers_license_helper -> #attempt_license_renewal or #validate_renewed_license
```